### PR TITLE
April updates

### PR DESCRIPTION
**Pull request by wwarner-salesforce**
goes with https://github.com/mulesoft/docs-release-notes/pull/670

### CODE REVIEW

**Comment by wwarner-salesforce** [comment]
Page name validation info is from EXC-3041.

**Review by kuntald — COMMENTED** [review]
Looks good to me. I'd wait for @mcampo 's review, before moving forward.

**Review by mcampo — APPROVED** [review]

File: modules/ROOT/pages/to-describe-an-asset.adoc [hunk 1]
```diff
@@ -49,6 +49,8 @@ In Markdown, use this syntax to reference a web served image:
 
 In Edit mode, you can reorder pages by clicking the page name and dragging the page up or down in the left nav bar.
 
+All page names are validated. Page names cannot start with `.`, cannot contain the characters `@`, `*`, `+`, `/`, `_`, or `\`, cannot be the reserved name `home`, cannot be the name of an existing page, and must be from 1 to 128 characters.
```
Comment (valkyrie69) [inline modules/ROOT/pages/to-describe-an-asset.adoc]: I would do a bullet list just to make it cleaner, but it's a minor thing and you can take it or leave it. For example:
Page names must be from 1 to 128 characters in length. 
Page names cannot:
- Start with...
- Contain the characters...
- Use the reserved name `home`
- Be the name of an existing page

**Review by valkyrie69 — APPROVED** [review]

File: modules/ROOT/pages/to-describe-an-asset.adoc (hunk 1 above)
Comment (wwarner-salesforce) [reply modules/ROOT/pages/to-describe-an-asset.adoc]: Yes, I agree. Switched it to a bullet list. Thank you!

**Pull request merged**